engine/settings/celery_task_routes.py

```diff
@@ -11,8 +11,7 @@
     "apps.alerts.tasks.wipe.wipe": {"queue": "default"},
     "common.oncall_gateway.tasks.create_oncall_connector_async": {"queue": "default"},
     "common.oncall_gateway.tasks.delete_oncall_connector_async": {"queue": "default"},
-    "common.oncall_gateway.tasks.create_slack_connector_async": {"queue": "default"},
-    "common.oncall_gateway.tasks.delete_slack_connector_async": {"queue": "default"},
+    "common.oncall_gateway.tasks.create_slack_connector_async_v2": {"queue": "default"},
```

Comment:
right now we're not processing `common.oncall_gateway.tasks.create_slack_connector_async_v2` tasks (afaict)